docs/tutorials/distributed/distributed_slowfast.py

```diff
@@ -82,26 +63,51 @@
 #
 #     ssh-rsa XXXXXXXXXXXXXXXX node1@ip-123-123-1-123
 #
-# The content in the middle is the actuall ssh key, and the IP address in the end is the node's internal IP address.
+# The content in the middle is the actuall ssh key for node1, and the IP address in the end is the node1's internal IP address.
+# Similarly, ssh into other machines (node2, node3 and node4) and perform the same steps to generate the ``id_rsa.pub`` file for each machine.
```

Comment:
what about:
- generate `id_rsa` and `id_rsa.pub` once on node1
- copy both file to all nodes
- for each of the node, append authorized_keys with content of `id_rsa.pub`
?